extensions/data-plane/data-plane-aws-s3/src/test/java/org/eclipse/edc/connector/dataplane/aws/s3/S3DataPlaneIntegrationTest.java

```diff
@@ -105,6 +107,12 @@ void should_copy_using_destination_object_name_case_single_transfer(List<String>
         var objectNameInDestination = "object-name-in-destination";
         var objectContent = UUID.randomUUID().toString();
 
+        //Put folder 0 byte size file marker. AWS does this when a folder is created via the console.
+        if (!isSingleObject) {
```

Comment:
I know the test was already structured like this, but it's not a good practice to have conditionals into a test, better to have 2 distinct tests. It could also be solved in a different PR